boards/arm/nucleo_wb55rg/doc/nucleowb55rg.rst

```diff
@@ -190,6 +192,10 @@ Default Zephyr Peripheral Mapping:
 - LD1 : PB5
 - LD2 : PB0
 - LD3 : PB1
+- SPI_1_NSS : PA4 (arduino_spi)
```

Comment:
The commit message says, "enables SPI1 and SPI2", but only looks like SPI1 is documented?

Reply:
I can change it if you'd like.  For the SOC, I added both SPI1 and SPI2 with all the different configurations in pinmux. For the nucleo_wb55rg dev board, I only enabled one configuration of SPI1, which is the one I documented.  Should I be enabling everything on a dev board as well?

Reply:
If only SPI1 is enabled on the board, then the docs are just fine (I was thinking it was an oversight in the documentation).